t/23-baseclass.t

```diff
@@ -892,4 +892,8 @@ subtest 'child process handling' => sub {
     combined_like { $baseclass->_stop_children_processes } qr/waitpid for $pid returned/, 'stopped child again';
 };
 
+subtest 'deprecate backend' => sub {
+    throws_ok { backend::baseclass::handle_deprecate_backend('AMT') } qr/is unsupported and planned to be\nremoved from os-autoinst eventually/, 'deprecated message is displayed';
```

Comment:
This is gone now, though? I'm confused why you're adding this test 🤔

Reply:
the code coverage was failing. Likely because uncover `handle_deprecate_backend` somehow

Reply:
Ah. I guess it was only used used with amt before so now it's not implicitly covered anymore. Makes sense!

Reply:
I guess this is related: https://app.codecov.io/gh/os-autoinst/os-autoinst/pull/2573/blob/backend/baseclass.pm#L1367